snuba/cli/entities.py

```diff
@@ -60,3 +62,12 @@ def describe(entity_name: str) -> None:
         entity.describe().accept(CLIDescriber())
     except InvalidEntityError:
         click.echo(f"Entity {entity_name} does not exists or it is not registered.")
+
+
+@entities.command()
```

Comment:
this is a script with a purpose, can you put its purpose in a comment in this file?

Reply:
added